torch_em/model/unetr.py

```diff
@@ -279,21 +324,12 @@ def forward(self, x):
         return self.block(x)
 
 
-def build_unetr_with_sam_intialization(out_channels=1, model_type="vit_b", checkpoint_path=None):
-    if get_sam_model is None:
-        raise RuntimeError(
-            "micro_sam is required to initialize the UNETR image encoder from segment anything weights."
-            "Please install it from https://github.com/computational-cell-analytics/micro-sam"
-            "and then rerun your code."
-        )
-    predictor = get_sam_model(model_type=model_type, checkpoint_path=checkpoint_path)
-    _image_encoder = predictor.model.image_encoder
+def build_unetr_with_sam_initialization(out_channels=1, model_type="vit_b", checkpoint_path=None):
```

Comment:
We can get rid of these functions. The current design is simple enough and we don't need to wrap it in a function.

Reply:
Okie. In that case, I will document it in the directory the correct usage for different ViT models with and without initialization.